src/host/outputStream.hpp

```diff
@@ -88,6 +88,8 @@ class ConhostInternalGetSet final : public Microsoft::Console::VirtualTerminal::
                                     const bool fIsForeground) override;
 
     BOOL PrivateBoldText(const bool bolded) override;
+    BOOL PrivateGetExtendedTextAttributes(ExtendedAttributes* const pAttrs) override;

```

Comment:
Why not just a ref? Matching existing pattern?

Reply:
That's correct, I was just trying to match the existing interface here. It's a little dated sure, but it's _consistent_